routes/booking.go

```diff
@@ -208,9 +208,8 @@ func (app *App) GetBookingsByUserID(w http.ResponseWriter, r *http.Request) {
 }
 
 func (app *App) GetBookingsByDateRange(w http.ResponseWriter, r *http.Request) {
-	queryParams := r.URL.Query()
-	start := queryParams["start"][0]
-	end := queryParams["end"][0]
+	start := r.FormValue("start")
```

Comment:
can i ask why we switched from queryParams to FormValue?

Reply:
This is actually a good point and I'm not sure if I addressed this properly. I'll talk to you about it quickly after the meeting